portfolio/src/main/java/com/google/sps/data/ScheduledTask.java

```diff
@@ -11,6 +11,7 @@
   private final Task task;
   private final Instant startTime;
   private Optional<SchedulingCompleteness> schedulingCompleteness;
+  private Optional<Integer> schedulingCompletenessInt;
```

Comment:
Curious why we need this? Whenever you need to reference this int value can't you just call `schedulingCompleteness.get().getValue()`? Keeping them separate is more complicated and makes it easier to make a mistake where you get a mismatch between the `SchedulingCompleteness` and the `Integer`.

Reply:
Talked offline about this. I see now, it's because you want to serialize this object and send it to the client and have them work with the int value. Usually in this case you'd explicitly specify how you want the serialization happen so that the server only sees the enum and the client only sees the int, but maybe that's not straightforward with Gson. Okay that's fine you can leave it as is.